docs/installation.md

```diff
@@ -62,10 +62,10 @@ The following instructions assume that you have local AWS credentials in `~/.aws
 
 ### 1. Clone this repository
 
-The mainline branch of this repository develops rapidly. If you want to obtain the latest stable copy of the code, clone this repository using the tag of the most recent release, which currently is 1.0.4.
+Clone this repository to obtain the latest stable copy of the code:
 
 ```bash
-git clone -b 1.0.4 https://github.com/awslabs/performance-dashboard-on-aws.git
+git clone -b https://github.com/awslabs/performance-dashboard-on-aws.git
```

Comment:
Should it be just git clone https://github.com/awslabs/performance-dashboard-on-aws.git without -b?

Reply:
@yinxudeng my fault, removed the branch number but not the "-b"